custom_components/healthchecksio/__init__.py

```
@@ -87,15 +99,26 @@ async def async_unload_entry(
     hass: core.HomeAssistant, config_entry: config_entries.ConfigEntry
 ) -> bool:
     """Unload a config entry."""
-
-    unload_ok = await hass.config_entries.async_forward_entry_unload(
-        config_entry, Platform.BINARY_SENSOR
-    )
+    _LOGGER.debug(f"Unloading Config Entry: {config_entry.as_dict()}")
```

Ruff (0.11.9)

Logging statement uses f-string

(G004)